packages/docusaurus-theme-classic/src/validateThemeConfig.ts

```diff
@@ -202,6 +202,7 @@ const LocaleDropdownNavbarItemSchema = NavbarItemBaseSchema.append({
 
 const SearchItemSchema = Joi.object({
   type: Joi.string().equal('search').required(),
+  className: Joi.string().default(''),
```

Comment:
```suggestion
  className: Joi.string(),
```

I'd remove the default value?

For consistency with the Html item also having a className + the TS declaration where className is optional